app/WebParsing/UtsgJsonParser.hs

```diff
@@ -42,17 +31,53 @@ insertAllMeetings org = do
     manager <- liftIO $ newManager tlsManagerSettings
     response <- liftIO $ httpLbs request' manager
     let respBody = responseBody response
+    return respBody
```

Comment:
you can merge this with the previous line by just doing `return $ responseBody response`